README.md

```diff
@@ -2,12 +2,16 @@
 
 The Arcaflow Engine allows you to run workflows using container engines, such as Docker or Kubernetes. The plugins must be built with the [Arcaflow SDK](https://arcalot.io/arcaflow/creating-plugins/python/).
 
+## Pre-build binaries
```

Comment:
The wording here with `build` instead of `built` give me the impression that the instructions below talk about how to pre-build them rather than how to use the already pre-built binaries.

So I think that should be changed to `Pre-built binaries` or `Using pre-built binaries`